modules/engine/src/lib/model.js

```diff
@@ -403,7 +403,7 @@ export default class Model {
     }
   }
 
-  _checkProgram(shaderCache = null) {
+  _checkProgram() {
```

Comment:
Not used?

Reply:
Not used, yeah. I also just found a private program-manager var that is unused. Should I do a seperate PR for these, or fine here?

Reply:
A separate PR would be nice, but I can live with having it here, just needs extra back and forth to confirm why this is happening, so having it in one PR may not save much time in the end.